calculator/main.py

```diff
@@ -58,18 +81,25 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    records = []
+    for item in calc['history']:
+        records.append(item)
+        
+    return records
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'] = []
 
 
 def repeat_last_operation(calc):
     """
     Returns the result of the last operation executed in the history.
     """
-    pass
+    #First: retrieve info about the last informat
+    if len(calc['history']) > 0:
+        _, operation, params, _ = calc['history'][-1]
+        return perform_operation(calc, operation, params)
```

Comment:
You're already storing the result of the operation in your history so you can just return that result here. (ie. `return calc['history'][-1][-1]`) It doesn't really matter here as we're only performing simple operations, but it's something to keep in mind as since this is extensible it could potentially be performing complex and cpu-expensive operations that you don't necessarily want to have to compute again if you only want the result from the last time it was performed.